scripts/release/qemu-install.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
```

Comment:
Could you add a doc comment to this script to explain a bit what it does? I feel like I understand it now, but I think a small comment at the beginning would make this clearer, also why this installs rust's toolchain using rustup here :)